### PR TITLE
WIP, TEST: set xcode to 10 via input variable

### PR DESCRIPTION
**Pull request by mattip**
Try differently to set xcode to 10

### CODE REVIEW

**Comment by tylerjereddy** [comment]
@chrisrpatterson Could you advise us on this? It seems our Linux and Mac jobs are being sent to the Windows Machine node pool & ignoring our selection of a different platform, perhaps because of recent changes on the Azure side?

**Comment by tylerjereddy** [comment]
see also a recent SciPy PR which has a similar problem -- everything goes to an Ubuntu image irrespective of platform specification: https://github.com/scipy/scipy/pull/9523

File: azure-pipelines.yml [hunk 1]
```diff
@@ -51,10 +51,7 @@ jobs:
       versionSpec: '3.6'
       addToPath: true
       architecture: 'x64'
-  # NOTE: do we have a compelling reason to use older / newer
-  # versions of Xcode toolchain for testing?
-  - script: /bin/bash -c "sudo xcode-select -s /Applications/Xcode_10.app/Contents/Developer"
-    displayName: 'select Xcode version'
+      xcodeVersion: '10'
```
[the remark this answers is not in the document]
Comment (mattip) [reply azure-pipelines.yml]: Nice catch, I missed that the failures are due to windows VMs being used. Feel free to close the PR

**Comment by chrisrpatterson** [comment]
@tylerjereddy Sorry to hear that I have directly pinged the engineering team to take a look.  Thanks for letting us know.

**Comment by chrisrpatterson** [comment]
@tylerjereddy the issues is realted to case sensitivity.  You have vmIMage and it should be vmImage.  Not sure why that has changed but we are looking at it.

**Pull request closed without merging**